### ISSUE TITLE
Unable to stream: ffmpeg error "Option stimeout not found"

### ISSUE THREAD

**Issue by r-sherwood** [SeydX/homebridge-camera-ui#187]
I'm not able to connect to a mjpeg camera stream through homekit and homebridge-camera-ui.

The preview in homebridge-ui > plugins > Homebridge Camera Ui > Settings > Camera > [camera name ] works well.
Also still images are well displayed through homekit devices (iOS, macOS).

But I'm not able to get a working video stream on homekit devices and the camera.ui web interface.

homebridge-camera-ui v4.2.7 

Used camera:
```
esp32-cam
Tasmota firmware 9.5.0.2 (outputs a mjpeg stream, and still images)
```

My Configuration:
```
{
            "name": "CameraUI",
            "port": 8181,
            "theme": "auto",
            "language": "auto",
            "debug": true,
            "mqtt": {
                "active": false
            },
            "http": {
                "active": false,
                "localhttp": false
            },
            "ssl": {
                "active": false
            },
            "cameras": [
                {
                    "name": "ESP32-Cam",
                    "manufacturer": "espressif",
                    "model": "ESP32-Cam",
                    "motion": true,
                    "doorbell": true,
                    "switches": true,
                    "motionTimeout": 0,
                    "unbridge": false,
                    "videoConfig": {
                        "source": "-re -f mjpeg -i http://XXX.XXX.XXX.XXX:81/stream",
                        "stillImageSource": "-f mjpeg -i http://1XXX.XXX.XXX.XXX:81/cam.mjpeg",
                        "maxStreams": 1,
                        "maxWidth": 0,
                        "maxHeight": 0,
                        "maxFPS": 0,
                        "debug": true
                    }
                }
            ],
            "platform": "CameraUI"
        }
```

Error output:
```
[Plugin] ESP32-Cam: Video stream requested: 1280x720, 30 fps, 299 kbps
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: Starting video stream: nativexnative, native fps, 299 kbps
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: Stream command: /usr/lib/node_modules/homebridge-camera-ui/node_modules/ffmpeg-for-homebridge/ffmpeg -re -f mjpeg -stimeout 10000000 -i http://XXX.XXX.XXX.XXX:81/stream -an -sn -dn -codec:v libx264 -pix_fmt yuv420p -color_range mpeg -f rawvideo -preset ultrafast -tune zerolatency -b:v 299k -payload_type 99 -ssrc 13216603 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params 7CcC+pMGet5+XuAmwZIzhE+VXhrSh7fu0Twt1Z5s srtp://XXX.XXX.XXX.XXX:49690?rtcpport=49690&pkt_size=1316 -loglevel level+verbose -progress pipe:1
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info] ffmpeg version 4.3.1 Copyright (c) 2000-2020 the FFmpeg developers
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   built with gcc 6.3.0 (Raspbian 6.3.0-18+rpi1+deb9u1) 20170516
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   configuration: --pkgconfigdir=/build/workspace/lib/pkgconfig --prefix=/build/workspace --pkg-config-flags=--static --extra-cflags=-I/build/workspace/include --extra-ldflags=-L/build/workspace/lib --extra-libs='-lpthread -lm' --enable-static --disable-debug --disable-shared --disable-ffplay --disable-doc --enable-openssl --enable-gpl --enable-version3 --enable-nonfree --enable-pthreads --enable-libvpx --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libx264 --enable-runtime-cpudetect --enable-libfdk-aac --enable-avfilter --enable-libopencore_amrwb --enable-libopencore_amrnb --enable-filters --enable-decoder=h264 --enable-network --enable-protocol=tcp --enable-libspeex --enable-demuxer=rtsp --enable-omx-rpi --enable-mmal --enable-indev=alsa --enable-outdev=alsa
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   libavutil      56. 51.100 / 56. 51.100
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   libavcodec     58. 91.100 / 58. 91.100
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   libavformat    58. 45.100 / 58. 45.100
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   libavdevice    58. 10.100 / 58. 10.100
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   libavfilter     7. 85.100 /  7. 85.100
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   libswscale      5.  7.100 /  5.  7.100
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   libswresample   3.  7.100 /  3.  7.100
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [info]   libpostproc    55.  7.100 / 55.  7.100
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [tcp @ 0x362fe00] [verbose] Starting connection attempt to XXX.XXX.XXX.XXX port 81
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [tcp @ 0x362fe00] [verbose] Successfully connected to XXX.XXX.XXX.XXX port 81
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: [fatal] Option stimeout not found.
[13/07/2021, 20:33:22] [CameraUI] [Plugin] ESP32-Cam: ffmpeg exited with code: 1 and signal: null (Error)
[13/07/2021, 20:33:22] [CameraUI] [Sessions] ESP32-Cam removed from active sessions
[13/07/2021, 20:33:22] [CameraUI] [Sessions] Currently active streams: {"ESP32-Cam":{"activeStreams":0,"maxStreams":1}}
```

Guess it's an ffmpeg configuration/option issue. 
From where is the unknown option "-stimeout 10000000" coming from and how to remove or change it? Don't know any ffmpeg options like this.

**Comment by r-sherwood** [comment]
BTW no issues using homebridge-camera-ffmpeg

```
14/07/2021, 11:40:10] [Camera FFmpeg] [ESP32-CAM] Stream command: /usr/lib/node_modules/homebridge-camera-ffmpeg/node_modules/ffmpeg-for-homebridge/ffmpeg -f mjpeg -i http://XXX.XXX.XXX.XXX:81/stream -an -sn -dn -codec:v libx264 -pix_fmt yuv420p -color_range mpeg -r 10 -f rawvideo -preset ultrafast -tune zerolatency -filter:v scale='min(1280,iw)':'min(720,ih)':force_original_aspect_ratio=decrease,scale=trunc(iw/2)*2:trunc(ih/2)*2 -b:v 299k -payload_type 99 -ssrc 13791901 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params jPjVApP9vOTrvIp1FbqC+Y2IAKYoxr9M7tK4GFed srtp://XXX.XXX.XXX.XXX:50425?rtcpport=50425&pkt_size=1316 -loglevel level+verbose -progress pipe:1
```

**Comment by SeydX** [comment]
`-stimeout` is added automatically to prevent ffmpeg hangouts

it seems that i have to make it user dependent

**Comment by r-sherwood** [comment]
I've removed that option from config.start.js in my setup and it streams now great. 
Looking forward for the next release.

**Comment by SeydX** [comment]
Fixed in v4.2.8 (added new option to config.json > videoConfig > `stimeout`)
